Source/JavaScriptCore/runtime/JSObject.cpp

```diff
@@ -40,6 +40,7 @@
 #include "PropertyDescriptor.h"
 #include "PropertyNameArray.h"
 #include "ProxyObject.h"
+#include "SuperSampler.h"
```

Comment:
This is not needed, right?  Can  we remove it?

Reply:
Removed.